independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/UpdateProjectCommandHandler.java

```diff
@@ -78,11 +87,15 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
                             QuarkusProjectHelper.artifactResolver(), request);
                     invocation.log().info("Project update recipe has been created:\n%s", tempFile.toString());
 
-                    invocation.log().info("The command to update this project: \n" +
-                            "mvn org.openrewrite.maven:rewrite-maven-plugin:4.39.0:run \\\n" +
-                            "  -Drewrite.configLocation=%s \\\n" +
-                            "  -DactiveRecipes=%s -e",
-                            tempFile.toString(), RECIPE_IO_QUARKUS_OPENREWRITE_QUARKUS);
+                    invocation.log().info("The commands to update this project: \n" +
```

Comment:
Why not use Text Block here?

Reply:
Because we are still compiling Quarkus with Java 11.